### DISCUSSION TITLE
Unsubscribe static events handlers

### DISCUSSION THREAD

**Discussion by os-lmo** [AvaloniaUI/Avalonia#5421] [Q&A] [answered]
Hi,

Some Avalonia controls have static events which can be globally subscribed through the AddClassHandler method. For insrance, we can subcribe the MenuOpenedEvent - http://reference.avaloniaui.net/api/Avalonia.Controls/MenuBase/8450CD04 - using this method - http://reference.avaloniaui.net/api/Avalonia.Interactivity/RoutedEvent/C7C528DD.

My question is how can I unsubscribe these static events?

**Comment by jp2masa** [comment, marked as answer]
`AddClassHandler` returns `IDisposable`, which can be used to dispose the subscription.

**Reply by os-lmo** [reply]
Thanks
